### PR TITLE
#161215691 setup app structure 

### PR DESCRIPTION
**Pull request by danuluma**
#### What does this PR do?
It set up the app structure

#### Description of Task to be completed?
Set up the app structure
Create a test endpoint 

#### How should this be manually tested?
* This project requires python3 installed. Please install it from https://www.python.org/downloads/
if you don't have it already.
* download https://github.com/danuluma/dannstore/archive/ch-app-structure-161215691.zip
* extract using your favourite archive manager to any location on your computer
* Alternatively, clone the repo by pasting ```git clone https://github.com/danuluma/dannstore.git``` into your terminal or cmd (you need to have git installed if on windows)
* in a terminal, navigate to the extracted\cloned folder location
* run the app by typing ```python3 run.py``` into the terminal
* use postman to access http://localhost:5000/dann/api/v1/home 
* you should get a response in json format like this : 
```
{
    "message": "Hello, there ;-)"
}
```

#### What are the relevant pivotal tracker stories?
#161215691

#### Questions:

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+import os, sys
+
+LOCALPATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, LOCALPATH + '/../../../')
+
```
Comment (EdwardMudaida) [inline run.py]: What's the purpose of line 4 &  5?

File: app/tests/v1/test_v1.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+import unittest
+import json
+
+#local
+LOCALPATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, LOCALPATH + '/../../../')
+
```
Comment (ghost) [inline app/tests/v1/test_v1.py:9]: @danuluma What's the purpose of line 7 & 8?

File: run.py (hunk 1 above)
Comment (ghost) [inline run.py:6]: @danuluma What's the purpose of line 4 & 5?

File: app/tests/v1/test_v1.py (hunk 2 above)
Comment (danuluma) [reply app/tests/v1/test_v1.py:9]: it sets the root of the project to where the app folder(?module) is....It basically takes the current location and moves back 3 times(prepends with ```/../../../``` ) then sets that as the root(project root/base)....thus for any module I import here, I only have to specify its location from the project base ie in relation to this root path,,,,, I had problems when importing modules whereby modules were not being found (local imports)

Comment (danuluma) [reply app/tests/v1/test_v1.py:9]: and after researching a bit, that was the solution I got,,,and it worked

File: run.py (hunk 1 above)
Comment (danuluma) [reply run.py:6]: it's similar to the above ......

**Pull request merged**